nat-lab/tests/utils/connection_util.py

```diff
@@ -204,15 +204,24 @@ async def new_connection_raw(tag: ConnectionTag) -> AsyncIterator[Connection]:
     if tag in DOCKER_SERVICE_IDS:
         async with Docker() as docker:
             async with container_util.get(docker, container_id(tag)) as connection:
-                yield connection
+                try:
```

Comment:
what's the point of this? :confused:

Reply:
Pylint started to cry about that. I'm not sure if it was due to the pipfile lock update or I was running a newer version of Python locall.

Reply:
Can we remove it and see if CI pylint cries about it now?

Reply:
done

Reply:
you can now check pylint job..

Probably I could add proper cleanup to the underlying member objects from `ConnectionManager` instead: `Connection`, `NetworkSwitcher` and `ConnectionTracker`

Reply:
Here's the run in case you missed it: https://github.com/NordSecurity/libtelio/actions/runs/9682309888/job/26715057683?pr=630

I'll now fix those errors to get ready to merge.